src/main/java/net/imglib2/RealRandomAccessibleRealInterval.java

```diff
@@ -43,6 +43,10 @@
  * A function over an n-dimensional real interval that can create a random
  * access {@link Sampler}.
  * </p>
+ * <p>
```

Comment:
This one I had no idea, and actually commented in the PR here that I wasn't sure. It certainly makes sense that an RRARI actually should/must be defined everywhere within its interval. This is also more consistent with RAI.